lagom14-scala-conductr-bundle-lib/src/test/scala/com/typesafe/conductr/bundlelib/lagom/scaladsl/ConductRServiceLocatorSpecWithEnv.scala

```diff
@@ -0,0 +1,122 @@
+package com.typesafe.conductr.bundlelib.lagom.scaladsl
+
+import java.net.{ InetSocketAddress, URI => JavaURI }
+
+import akka.actor._
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.headers.{ CacheDirectives, Location, `Cache-Control` }
+import akka.http.scaladsl.model.{ HttpEntity, HttpResponse, StatusCodes }
+import akka.http.scaladsl.server.Directives._
+import akka.stream.ActorMaterializer
+import akka.testkit.TestProbe
+import com.lightbend.lagom.internal.client.{ CircuitBreakerConfig, CircuitBreakerMetricsProviderImpl, CircuitBreakers }
+import com.lightbend.lagom.internal.spi.CircuitBreakerMetricsProvider
+import com.lightbend.lagom.scaladsl.api.{ Descriptor, ServiceAcl, ServiceInfo }
+import com.lightbend.lagom.scaladsl.client.CircuitBreakersPanel
+import com.typesafe.conductr.bundlelib.play.api.{ Env => PlayEnv }
+import com.typesafe.conductr.bundlelib.scala.{ URI, URL }
+import com.typesafe.conductr.lib.AkkaUnitTestWithFixture
+import play.api.libs.ws.ahc.AhcWSComponents
+import play.api.routing.Router
+import play.api.test.Helpers._
+import play.api.{ ApplicationLoader, BuiltInComponentsFromContext, Environment }
+
+import scala.collection.immutable
+import scala.concurrent.{ Await, Future }
+import scala.util.{ Failure, Success }
+
+class ConductRServiceLocatorSpecWithEnv extends AkkaUnitTestWithFixture("ConductRServiceLocatorSpecWithEnv") {
+
+  def systemFixture(f: this.FixtureParam) = new {
+    implicit val system = f.system
+    implicit val timeout = f.timeout
+    implicit val mat = ActorMaterializer.create(system)
+    val app = new BuiltInComponentsFromContext(ApplicationLoader.createContext(Environment.simple())) with AhcWSComponents with ConductRServiceLocatorComponents {
```

Comment:
I was suggested by @TimMoore to use `LagomApplication` instead of  `BuitlInComponentsFromContext` in the Lagom scala test: https://github.com/typesafehub/service-locator-dns/pull/17#issuecomment-331335946

Despite the suggestion was for `service-locator-dns`, I think this is also applicable to our `conductr-lib`. The example usage is as following:

https://github.com/typesafehub/service-locator-dns/pull/17/files#diff-e6c5f1d4e790368c0587bcefe7f22747R43

If you do modify `BuiltInComponentsFromContext` to `LagomApplication`, would you also do it for the `lagom13` scala please?

Reply:
Thanks - this is set.